src/Searchbar.js

```diff
@@ -11,6 +11,12 @@ export default class SearchBar extends Component {
                    <option value ='ascending'>Ascending</option>
                    <option value ='descending'>Descending</option>
                </select>
+               <select onChange = {this.props.sort2Change}>
```

Comment:
Nice work, Kevin! For those last 1.5 points, try refactoring your select tags into a separate component. I noticed that your Sort.js right now is a bit empty, but this should be an easy refactor